store/src/db/mod.rs

```diff
@@ -60,46 +61,46 @@ impl Db {
     }
 
     /// Loads all the nullifiers from the DB.
-    pub async fn get_nullifiers(&self) -> Result<Vec<(RpoDigest, BlockNumber)>, anyhow::Error> {
```

Comment:
to make it easier to understand what the sql methods are doing, I changed the prefixes to match the SQL statement. (this is useful to identify upserts vs inserts at a glance, the selects are just for consistency)